### PR TITLE
fix: Update notification messages for verifying account

### LINKED ISSUE #5516: Update notification messages for verifying account and show "full access" notification on frontpage
Please change notification messages as follows and change functionality as follows:
Screenshot from 2020-11-07 07-18-51

* [x] "Account unverified" notification
  - original notification
Your account is unverified. Please verify it by clicking on the confirmation link that has been emailed to you.
  - change to ->
Your account functionality is limited. To get full access to all features you need to verify your email by clicking on the confirmation link that has been emailed to you.

Screenshot from 2020-11-07 07-21-48

* [x] "full access" notification
  - original notification
Thank you, your email has been verified successfully!
  - change to ->
Thank you, your email has been verified successfully! You now have full access to all features.
* [ ] Instead of showing the "Full access - Thank you..." on a white page, show the front page with the message on top

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5516 

#### Short description of what this resolves:
update notification mesaages for verifying account.

Frist 2 points (changes) have done in this PR.

Working on third point(change).

__screenshots__

5516 a

5516 b

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/verify.hbs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {{#if this.success}}
   <div class="ui {{if this.isLoading 'loading'}} icon info message eight wide column center aligned">
-    <p>Your email has been verified successfully!{{#if (not this.session.isAuthenticated)}} Please login to continue. {{/if}}</p>
+    <p>your email has been verified successfully! You now have full access to all feature.{{#if (not this.session.isAuthenticated)}} Please login to continue. {{/if}}</p>
```
Comment (iamareebjamal) [inline app/templates/verify.hbs:3]: English sentences start with a capital letter

Comment (iamareebjamal) [inline app/templates/verify.hbs:3]: all is used for plurals

Comment (mariobehling) [reply app/templates/verify.hbs:3]: Please use: 
Your account is unverified. Please verify it by clicking on the confirmation link that has been emailed to you.

Comment (mariobehling) [reply app/templates/verify.hbs:3]: Please use:
Thank you, your email has been verified successfully! You now have full access to all features.

**Comment by sachinchauhan2889** [comment]
@iamareebjamal @mariobehling sir, i have deleted  _/verify_ route as it will be of no use if we want to display message at front page.
i have done all neccesary changes.

I am still working on this. One thing is still missing. After registration of new user, email have verification link as _localhost:4200/verify?token=abcdefcghtuheru__.  This route is removed in this PR.
Instead of this link we need to send this link _localhost:4200/?token=abcdefghijhkuh__. This link completes the verification work.
I think this mail can be manipulated in open-event-server. i am looking into open-event-server repo.
please test these changes. while clicking on verification link please remove _verify_ route and pass token in _index_ route.
please let me know if any other changes needed.

__screenshots__



333333   new

please remove /verify from confirmation link for testing.

**Comment by iamareebjamal** [comment]
That's not what we want. Please revert

**Comment by sachinchauhan2889** [comment]
@iamareebjamal sir, instead of showing message on top of front page. You need front page content in this white space.
Am i thinking right sir.

**Comment by sachinchauhan2889** [comment]
@iamareebjamal sir, can u tell be little bit more about desired result, so that i can think in right direction. Do we want front page content in white space area??

**Comment by iamareebjamal** [comment]
> i have deleted /verify route as it will be of no use

It will definitely be of use, so please revert this

**Comment by sachinchauhan2889** [comment]
@iamareebjamal @mariobehling sir, i have revert my changes.
sir can you please elaborate third point. 
want full-access message on front page i.e. index page
         or
message remains at /verify route, and want front page index template to be below message inplace of white space.

**Comment by sachinchauhan2889** [comment]
> @iamareebjamal @mariobehling sir, i have revert my changes.
> sir can you please elaborate third point.
> want full-access message on front page i.e. index page
> or
> message remains at /verify route, and want front page index template to be below message inplace of white space.

@iamareebjamal sir, have a look at this.

**Review by iamareebjamal — APPROVED** [review]

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Build is failing

**Comment by iamareebjamal** [comment]
Please always fix the build before asking for reviews.

**Comment by sachinchauhan2889** [comment]
@iamareebjamal sir, done. Build successful.

**Comment by iamareebjamal** [comment]
Are we looking at the same PR?

image

**Comment by sachinchauhan2889** [comment]
@iamareebjamal  sorry sir, it was opened in browser.

**Review by snitin315 — CHANGES_REQUESTED** [review]
need to update tests. 
https://github.com/fossasia/open-event-frontend/blob/6d509ce2e2b68ed85632eda04dfe97dac16ce5e9/tests/integration/components/unverified-user-message-test.js#L37-L44

**Comment by sachinchauhan2889** [comment]
> need to update tests.

@snitin315 thank you, i am updating tests.

**Comment by sachinchauhan2889** [comment]
@iamareebjamal sir, test also updated. please review.

**Comment by iamareebjamal** [comment]
Not until build passes

**Review by snitin315 — APPROVED** [review]

**Pull request merged**